### ISSUE TITLE
PromQL rollup_candlestick - invalid high value

### ISSUE THREAD

**Issue by aleksraiden** [VictoriaMetrics/VictoriaMetrics#204]
**Describe the bug**
If fetched OHLC data using extended functions rollup_candlestick, we see that high value isn't always highest at the one candles. For example, after parse json, we have one candles: 
`open: 7779.29, high: 7777.205, low: 7776.6375, close: 7777.205`

Open value is the biggest then high.

**Screenshots**
[Screenshot from Grafana](https://contractmarketcap.com/images/grafana_vic_bug.png)
Query: api/v1/query_range?query=rollup_candlestick(test_test)&start=1569915000&end=1570519800&step=600

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
 build version: victoria-metrics-20190914-082936-tags-v1.27.3-0-g63f2b539
```

**Comment by valyala** [comment]
@aleksraiden , thanks for the bug report. It should be fixed in 956fdd89d3f1bd4045152fae271204add655b3cf  . The bug fix will be available in the next release - `v1.28.0`.

**Comment by valyala** [comment]
The bugfix is available in [v1.28.0-beta6](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). @aleksraiden , could you verify the issue is fixed there?

**Comment by valyala** [comment]
Closing the issue as fixed. Feel free re-opening it if new issues with `rollup_candlestick` (aka OHLC) will  arise.
